configs/approval.json

```diff
@@ -113,9 +131,41 @@
           "resourceDefinitions": [
             {
               "attributeFilter": {
-                "value": "*",
-                "key": "id",
-                "operation": "equal"
+                "key": "scope",
+                "operation": "equal",
+                "value": "admin"
+              }
+            }
+          ]
+        }
+      ]
+    },
+    {
+      "name": "Approval User",
+      "system": true,
+      "version": 4,
+      "description": "An approval user role which grants permissions to create/read/cancel a request.",
```

Comment:
`.... /cancel a request, and read workflows." `

Please add read action permission.